harness/determined/util.py

```diff
@@ -415,6 +417,15 @@ def forward_signals(p: subprocess.Popen, *signums: signal.Signals) -> Iterator[N
         signums = tuple(getattr(signal, name) for name in names if hasattr(signal, name))
 
     def signal_passthru(signum: Any, frame: Any) -> None:
+        # Forward the signal to the target process and all its children
+        # On HPC systems, not all children shutdown.
+        try:
+            children = psutil.Process(p.pid).children(recursive=True)
+            for cp in children:
+                cp.send_signal(signum)
+        except psutil.NoSuchProcess as err:
```

Comment:
What about other exceptions?

Reply:
I guess the question is what other errors could occur.  I guess I could ignore them all.

Reply:
Right but the existing send_signal  call didn't handle any signals.   We have the same potential ones there.
The potential ones from psutils.Process are ZombieProcess, AccessDenied, TimeoutExpired.
I changed it to ignore/log all exceptions, but unclear that is always the best response.  In this case I guess it would avoid raising any signal that wasn't previously rasised.